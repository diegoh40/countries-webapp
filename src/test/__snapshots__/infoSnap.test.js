import { Provider } from 'react-redux';
import { BrowserRouter, Switch, Route } from 'react-router-dom';
import { render, screen } from '@testing-library/react';
import store from '../../redux/configureStore';
import Info from '../../pages/Details';

/*
test('Info snapshot', () => {
    // eslint-disable-next-line testing-library/render-result-naming-convention
    const treee = render
      (
        <Provider store={store}>
          <BrowserRouter>
          <Switch>
            <Route>
            <Info />
            </Route>
            </Switch>
          </BrowserRouter>
        </Provider>,
      )

    expect(treee).toMatchSnapshot();
  });
 */

test('Info snapshot', () => {
  render(
    <Provider store={store}>
      <BrowserRouter>
        <Switch>
          <Route>
            <Info />
          </Route>
        </Switch>
      </BrowserRouter>
    </Provider>,
  );
  const linkElement1 = screen.getByText(/Capital/i);
  expect(linkElement1).toMatchSnapshot();
});
