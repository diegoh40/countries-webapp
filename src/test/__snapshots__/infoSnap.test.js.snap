// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info snapshot 1`] = `
<div>
  gg
</div>
`;
